src/mlpack/methods/kmeans/hamerly_kmeans_impl.hpp

```diff
@@ -156,21 +197,31 @@ double HamerlyKMeans<DistanceType, MatType>::Iterate(const arma::mat& centroids,
     }
   }
 
-  // Now update bounds (lines 3-8 of Update-Bounds()).
-  for (size_t i = 0; i < dataset.n_cols; ++i)
+  // Now update bounds
```

Comment:
Revert the comment please. It is extremely important for maintenance to be able to connect the code to the parts of the original paper.